builder/bundles.go

```diff
@@ -312,9 +312,27 @@ func repoPkgFromNoopInstall(installOut string) (repoPkgMap, repoRpmMap) {
 	return repoPkgs, repoRpmMap
 }
 
+func queryRpmName(packageCmd []string, pkgName string) string {
+	queryStringRpm := merge(
+		packageCmd,
+		"repoquery",
+		"--location",
+		"--repo local",
+	)
+	var name string
+	queryStringRpm = append(queryStringRpm, pkgName)
+	outBuf, _ := helpers.RunCommandOutputEnv(queryStringRpm[0], queryStringRpm[1:], []string{"LC_ALL=en_US.UTF-8"})
+	if outBuf.String() != "" {
+		rpm := strings.Split(outBuf.String(), "\n")
+		_, name = filepath.Split(rpm[0])
```

Comment:
Like what @rchiossi suggested, remove `var name string` and just use `_, name :=` here.